README.md

```diff
@@ -52,7 +52,7 @@ Thank you for considering and taking the time to contribute! Before contributing
 
 ## 📌 Credits :
 
-This project exists thanks to all the people who contribute.
+This project exists just because of you, thanks to all the people who contribute.
```

Comment:
change it to `This project exists, thanks to all the people who contribute.`

Reply:
> change it to `This project exists, thanks to all the people who contribute.`

kindly check other issues opened by me as well as other pr opened